doc/build-helpers/trivial-build-helpers.chapter.md

```diff
@@ -557,14 +557,18 @@ This creates a derivation with a directory structure like the following:
 
 ## `writeReferencesToFile` {#trivial-builder-writeReferencesToFile}
 
-Writes the closure of transitive dependencies to a file.
+Deprecated. Use [`writeReferenceClosureToFile`](#trivial-builder-writeReferenceClosureToFile) instead.
+
+## `writeReferenceClosureToFile` {#trivial-builder-writeReferenceClosureToFile}
```

Comment:
Also, `writeReferencesClosureToFile` is a bit of a mouthful, I can imagine myself typoing or forgetting if references are plural